sdk/keyvault/keyvault-keys/package.json

```diff
@@ -27,35 +27,36 @@
-    "src/**/*.ts",
-    "rollup.config.js",
+    "types/src/",
+    "dist/",
+    "dist-esm/",
+    "src/",
     "tsconfig.json"
   ],
+  "browser": {
+    "./dist/index.js": "./browser/azure-keyvault-keys.min.js",
+    "./dist-esm/src/index.js": "./dist-esm/src/index.js",
+    "os": false,
+    "process": false
+  },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:minify": "uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js 2>&1",
     "build:samples": "tsc samples/helloWorld.ts",
-    "build:test": "npm run build",
-    "build": "npm run extract-api && npm run build:samples && rollup -c rollup.config.js 2>&1",
+    "build:browserzip": "gulp zip",
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
+    "build:test": "npm run build:es6 && rollup -c rollup.test.config.js 2>&1",
+    "build": "npm run extract-api && npm run build:samples && npm run build:es6 && npm run build:nodebrowser && npm run build:browserzip",
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"tests/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-esm test-dist types *.tgz *.log",
+    "clean": "rimraf dist dist-esm dist-test types *.tgz *.log browser statistics.html coverage && find src/ -type f -name '*.js' -exec rm {} + && find tests/ -type f -name '*.js' -exec rm {} +",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"tests/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "ts-mocha -p tests/tsconfig.test.json tests/*.test.ts tests/**/*.test.ts --timeout 1200000 --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha --require source-map-support/register --reporter mocha-multi --timeout 1200000 --reporter-options spec=-,mocha-junit-reporter=- --full-trace -t 120000 dist-test/index.node.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint -c ../../.eslintrc.json src tests samples --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint -c ../../.eslintrc.json src tests samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o keyvault-keys-lintReport.html || exit 0",
+    "lint:fix": "eslint --rule 'no-invalid-this: 0' -c ../../.eslintrc.json src tests samples --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint --rule 'no-invalid-this: 0' -c ../../.eslintrc.json src tests samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o keyvault-keys-lintReport.html || exit 0",
```

Comment:
Added this rule since I'm using `this.title` and other properties of Mocha's test prototypes in my tests.

Reply:
I don't like having special lint rules for specific packages.  Why is this rule specific to this package?  Should the test code for this package instead be modified to align with other packages which don't need this special rule?

Reply:
Fixing it!